draft-ietf-rats-corim.md

```diff
@@ -1917,37 +1972,37 @@ Corroboration is the process of determining whether actual Attester state (as co
 If satisfied, the RVP authority is added to the matching ACS entry.
 
 Reference Values are matched with ACS entries by iterating through the `rv` list.
-For each `rv` entry, the `condition` ECT is compared with an ACS ECT.
+For each `rv` entry, the `condition` ECT is compared with an ACS ECT, where the ACS ECT `cmtype` contains `evidence`.
```

Comment:
Can one party's reference values not be for another party's endorsements?

Reply:
> Can one party's reference values not be for another party's endorsements?

Structurally, that would be reasonable since both Evidence and Endorsements are expressions of actual state. However, I don't know why it is necessary to model a condition on endorsements using RVs since a conditional endorsement condition could be expressed using possible-state expressions. I'll review the current wording, but I believe the cmtype usage is specific to the addition ECTs and silent on the condition ECTs.

Reply:
Added an issue and tracking annotation.